### ISSUE TITLE
FR: Whitelist to allow Blocked items for specific domains

### ISSUE THREAD

**Issue by peterlaws** [andryou/scriptsafe#83]
I'd like to be able to allow blocked items for each domain, similar to Ghostery.

Let's say you block adsense by default, I'd like to allow it on a domain, eg. mydomain.com

I know I can temporary disable SS or temporary allow the resource, it was just a thought.

**Comment by Enigma0** [comment]
Yeah seconded. I don't think the right way to make a complete domain exception is to disable ScriptSafe completely for the session.
